README.md

```diff
@@ -49,10 +49,26 @@ grant the Service Principal
 and [generate an API token](https://docs.databricks.com/dev-tools/api/latest/token-management.html#operation/create-obo-token) on its behalf.
 
 ## Azure
-[//]: # (TODO: Add back steps to create an Azure Service Principal.)
-You can create a Personal Access Token from the `User Settings` page in the
-Databricks workspace and pass it to the action as an input.
-The generated token can be stored as a GitHub Actions secret named e.g. `MY_DATABRICKS_PERSONAL_TOKEN`.
+For security reasons, we recommend using a Databricks service principal AAD token.
+You can:
```

Comment:
```suggestion

### Create an Azure Service Principal
You can:
...
### Use the Service Principal in your GitHub Workflow
* Add the following snippet to the start of your GitHub workflow. This will create a new AAD token and save its value in the `DATABRICKS_TOKEN` environment variable for use in subsequent steps
(include YAML snippet here or link to one of the snippets below. Snippet can contain code comments explaining the resource ID).
```

I wonder if it'd help for clarity to separate the steps for creating the Azure SP and using it in the workflow. Also, I think this section is probably just best demonstrated through a code example

Reply:
Sounds Good! done